### PR TITLE
Integrate new TCN dataset and prediction

### PR DESCRIPTION
**Pull request by Purg**
Update to ROS side of things to make use of the extensive updated to the TCN submodule.

### CODE REVIEW

File: TRAIN_AND_RUN_README.md [hunk 1]
```diff
@@ -1,61 +1,132 @@
...
+Consult a team member for login information.
+
+Each task ("skill") has their own sub-page ("In-Lab Data" link) from which sets
+of data are described and referred to.
+Data is stored on their SFTP server, to which the "Click to Download" links
+refer to.
+
+Storage of downloaded ZIP archives, and their subsequent extractions, should
+follow the pattern.
+A script is provided
```
Comment (josephvanpeltkw) [inline TRAIN_AND_RUN_README.md:75]: recommend listing the location of the script for easy reference

File: tmux/demos/medical/Kitware-M2.yml [hunk 2]
```diff
@@ -115,9 +118,9 @@ windows:
       -p image_ts_topic:=PVFramesBGR_ts
       -p det_topic:=ObjectDetections2d
       -p pose_topic:=PatientPose
+      -p activity_config_file:=${CONFIG_DIR}/activity_labels/medical/m2.yaml
       -p model_weights:=${MODEL_DIR}/activity_classifier/m2_tcn.ckpt
-      -p model_mapping:=${MODEL_DIR}/activity_classifier/m2_mapping.txt
-      -p model_det_label_mapping:=${ANGEL_WORKSPACE_DIR}/config/object_labels/medical/m2.json
+      -p model_config:=${MODEL_DIR}/activity_classifier/m2_config.yaml
```
Comment (josephvanpeltkw) [inline tmux/demos/medical/Kitware-M2.yml:123]: Should we make these changes on the other medical yml files?

**Review by josephvanpeltkw — APPROVED** [review]

File: tmux/demos/medical/Kitware-M2.yml (hunk 2 above)
Comment (Purg) [reply tmux/demos/medical/Kitware-M2.yml:123]: I don't have any other models/configs for the other tasks, so it seemed to me that changing other configs would just be notional change. Maybe an alternative here is that don't just modify that existing config but create a new one. We would not be able to run the other configurations due to this API change, so I could move them to an "outdated" folder or something?

File: TRAIN_AND_RUN_README.md (hunk 1 above)
Comment (Purg) [reply TRAIN_AND_RUN_README.md:75]: I see I forgot to finish that sentence.

**Pull request merged**